api/IDBFactory.json

```diff
@@ -0,0 +1,600 @@
+            "deprecated": false
+          }
+        },
+        "available_in_workers": {
+          "__compat": {
+            "description": "Available in workers",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "37"
+              },
+              "firefox_android": {
+                "version_added": "37"
+              },
+              "ie": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            }
+          }
+        },
+        "idb_2": {
+          "__compat": {
+            "description": "Indexed Database 2.0",
+            "support": {
+              "webview_android": {
+                "version_added": "58"
+              },
+              "chrome": {
+                "version_added": "58"
+              },
+              "chrome_android": {
+                "version_added": "58"
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": null
+              },
+              "firefox_android": {
+                "version_added": null
+              },
+              "ie": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": "45"
+              },
+              "opera_android": {
+                "version_added": "45"
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            }
+          }
+        }
+      },
+      "deleteDatabase": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBFactory/deleteDatabase",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
```

Comment:
Copy this information to webview_android and chrome_android.

Reply:
👍